solr/core/src/test-files/solr/collection1/conf/solrconfig-tieredmergepolicyfactory.xml

```diff
@@ -28,6 +28,7 @@
       <int name="maxMergeAtOnce">19</int>
       <int name="maxMergeAtOnce">7</int>
       <int name="segmentsPerTier">9</int>
+      <int name="targetSearchConcurrency">1</int>
```

Comment:
Tests using this `solrconfig.xml` will fail with this addition here until we've upgraded to use Lucene 9.12 (which is not yet released).